packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -195,6 +195,17 @@ export const ApplicationMenu = (props: ApplicationMenuProps) => {
     ? getMenuVisibilitiesOfSubmenus(props.menu)
     : getMenuVisibilityOfMainmenu(props.menu);
 
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLLIElement>) => {
+    const currentlyFocusedItem = submenuRef.current?.querySelector(':focus');
+
+    if (e.key === 'Enter') {
+      setIsSubmenuFocused(true);
+      if (!currentlyFocusedItem) {
+        submenuRef.current?.querySelector('a')?.focus();
+      }
+    }
+  };
```

Comment:
Just to double check regarding my previous comment:

> Additionally, I've encountered what seems to be a bug. Pressing <kbd>Enter</kbd> on any menu item results in submenu items always being focusable.

What we're doing in this handler - once the user presses <kbd>Enter</kbd> `isSubmenuFocused` is set to `true` for good and this state never changes again. Which means that once user presses <kbd>Enter</kbd> the navigation works as previously (so the user has to navigate through all the menu and submenu items).

Is this expected and are we ok with that?

Reply:
No it is not the expected behaviour, this is a bug. Sorry, I thought I addressed this in the initial comment you made. I understand you now. Will look into it. 
Thank you.

Reply:
Fixed here: https://github.com/commercetools/merchant-center-application-kit/pull/3482/commits/4e07032305fa1fd7a4aa2a4655e77c65b2a8ad5c
Thank you